internal/proxy/proxy.go

```diff
@@ -7,14 +7,24 @@ import (
 	"net/http"
 	"net/url"
 
+	"github.com/hashicorp/vault-lambda-extension/internal/config"
 	"github.com/hashicorp/vault-lambda-extension/internal/vault"
 	"github.com/hashicorp/vault/sdk/helper/consts"
 )
 
+const (
+	VaultCacheControlHeaderName = "X-Vault-Cache-Control"
+
+	// TODO: remove these definitions once they're available in Vault's api package
```

Comment:
Does this need to be addressed before this PR is merged?

Reply:
I don't believe so.

Reply:
Added them to Vault's api package in https://github.com/hashicorp/vault/pull/14067